src/commands/config/reactionRole/set.ts

```diff
@@ -95,43 +91,43 @@ const command: Command = {
       }
     }
 
-    try {
-      const requestData: RoleReactionEvent = {
-        guild_id: msg.guild.id,
-        message_id: messageId,
-        reaction,
-        role_id: roleId,
-      }
+    const requestData: RoleReactionEvent = {
+      guild_id: msg.guild.id,
+      message_id: messageId,
+      reaction,
+      role_id: roleId,
+    }
 
-      const rrConfig: RoleReactionConfigResponse =
-        await config.updateReactionConfig(requestData)
-      if (rrConfig.success) {
-        message.react(requestData.reaction)
-        return {
-          messageOptions: {
-            embeds: [
-              composeEmbedMessage(msg, {
-                author: ["Reaction roles", msg.guild.iconURL()],
-                description: `Emoji ${requestData.reaction} is now setting to this role <@&${requestData.role_id}>`,
-              }),
-            ],
-          },
-        }
-      }
-    } catch (error) {
-      ChannelLogger.log(error as BotBaseError)
+    const res = await config.updateReactionConfig(requestData)
+    if (res.ok) {
+      message.react(requestData.reaction)
       return {
         messageOptions: {
           embeds: [
-            getErrorEmbed({
-              msg,
-              description:
-                "Role / emoji was configured, please type `$rr list` to check.",
+            composeEmbedMessage(msg, {
+              author: ["Reaction roles", msg.guild.iconURL()],
+              description: `Emoji ${requestData.reaction} is now setting to this role <@&${requestData.role_id}>`,
             }),
           ],
         },
       }
     }
+
+    if (res.error) {
+      ChannelLogger.log(new Error(res.error) as BotBaseError)
```

Comment:
```suggestion
throw new APIError({ message: msg, curl: res.curl, description: res.log })
```
tham khảo [verify command](https://github.com/consolelabs/mochi-bot/blob/develop/src/commands/community/verify/set.ts#L58) nha em